consensus/hotstuff/voteaggregator/vote_aggregator.go

```diff
@@ -2,390 +2,162 @@ package voteaggregator
 
 import (
 	"fmt"
+	"github.com/onflow/flow-go/engine"
```

Comment:
Let's move the new implementation to `v2/voteaggregator/vote_aggregator.go` for now, so that there would be less changes.

Once most are ready, we can start replacing by:
1. remove the existing `voteaggregator/vote_aggregator.go`, commit, and make PR
2. `git mv v2/voteaggregator/vote_aggregator.go voteaggregator/vote_aggregator.go`